bin/rbenv-update

```diff
@@ -1,23 +1,24 @@
 #!/usr/bin/env bash
-# Usage: rbenv update [--noop]
+# Usage: rbenv update [--noop|--quiet|--silent]
 # Summary: Updates to latest version of rbenv and plugins from git.
 set -e
 [ -n "$RBENV_DEBUG" ] && set -x
 
+# Parse command line flags
 noop=""
```

Comment:
Ah, `unset` is a bash built-in. Doesn't exist for fish, so probably best to leave as-is. (Kinda makes me sad, stylistically I would have liked `unset -v noop silent verbose`